process/graduation_criteria.adoc

```diff
@@ -29,6 +29,7 @@ To be accepted to incubating stage, a project must meet the sandbox stage requir
  * Demonstrate a substantial ongoing flow of commits and merged contributions.
  * Since these metrics can vary significantly depending on the type, scope and size of a project, the TOC has final judgement over the level of activity that is adequate to meet these criteria
  * A clear versioning scheme.
+ * Clearly documented security processes explaining how to report security issues to the project, and describing how the project provides updated releases or patches to resolve security vulnerabilities 
```

Comment:
Could we please showcase some existing / widely used process for a minimum bar here? For example the one from GH itself:

https://docs.github.com/en/code-security/security-advisories/adding-a-security-policy-to-your-repository

Reply:
when the sig-security folks write up their recommendations, we can switch the url around to what they come up with.

Reply:
i am fine with this merging as-is